### PR TITLE
Update services plugin

### PR DESCRIPTION
**Pull request by qmhu**
#### What type of PR is this?

Feature

#### What this PR does / why we need it:

Update services plugin

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: pkg/recommendation/recommender/idlenode/recommend.go [hunk 1]
```diff
@@ -33,23 +32,47 @@ func (inr *IdleNodeRecommender) Recommend(ctx *framework.RecommendationContext)
...
+		if memoryUsageUtilization > inr.memoryUsageUtilization {
+			return fmt.Errorf("Node %s is not a idle node, because the config value is %f, but the node memory usage utilization is %f ", ctx.Object.GetName(), inr.memoryUsageUtilization, memoryUsageUtilization)
+		}
 	}
 
 	// check if cpu request utilization lt config value
-	if cpuRequestUtilization := inr.getMaxValue(inr.cpuRequestUtilization, ctx.InputValue(cpuRequestUtilizationKey)); cpuRequestUtilization > inr.cpuRequestUtilization {
-		return fmt.Errorf("Node %s is not a idle node, because the config value is %f, but the node max cpu request utilization is %f ", ctx.Object.GetName(), inr.cpuRequestUtilization, cpuRequestUtilization)
+	if inr.cpuRequestUtilization != 0 {
+		cpuRequestUtilization, err := inr.BaseRecommender.GetPercentile(inr.cpuRequestUtilization, ctx.InputValue(cpuRequestUtilizationKey))
```
Comment (lbbniu) [inline pkg/recommendation/recommender/idlenode/recommend.go:58]: 第一参数应该是 inr.cpuPercentile

File: pkg/recommendation/recommender/idlenode/recommend.go [hunk 2]
```diff
@@ -33,23 +32,47 @@ func (inr *IdleNodeRecommender) Recommend(ctx *framework.RecommendationContext)
...
+		if cpuRequestUtilization > inr.cpuRequestUtilization {
+			return fmt.Errorf("Node %s is not a idle node, because the config value is %f, but the node cpu request utilization is %f ", ctx.Object.GetName(), inr.cpuRequestUtilization, cpuRequestUtilization)
+		}
 	}
 
 	// check if memory request utilization lt config value
-	if memoryRequestUtilization := inr.getMaxValue(inr.memoryRequestUtilization, ctx.InputValue(memoryRequestUtilizationKey)); memoryRequestUtilization > inr.memoryRequestUtilization {
-		return fmt.Errorf("Node %s is not a idle node, because the config value is %f, but the node max memory request utilization is %f ", ctx.Object.GetName(), inr.memoryRequestUtilization, memoryRequestUtilization)
+	if inr.memoryRequestUtilization != 0 {
+		memoryRequestUtilization, err := inr.BaseRecommender.GetPercentile(inr.memoryRequestUtilization, ctx.InputValue(memoryRequestUtilizationKey))
```
Comment (lbbniu) [inline pkg/recommendation/recommender/idlenode/recommend.go:69]: 第一个参数应该是 inr.memoryPercentile

Comment (qmhu) [reply pkg/recommendation/recommender/idlenode/recommend.go:69]: a mistake...

**Comment by qmhu** [comment]
@lbbniu PTAL

**Comment by lbbniu** [comment]
LGTM

**Pull request merged**